poetry.lock

```diff
@@ -1,72 +1,100 @@
 [[package]]
 name = "aiohttp"
-version = "3.7.4"
```

Comment:
I've never worked with `poetry` before, but it was telling me to run `poetry update` after I had changed the `importlib_metadata` requirement. The large diff in this file is the result. LMK if I need to run a different command instead.